src/Appwrite/Platform/Tasks/EdgeSync.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Appwrite\Platform\Tasks;
+
+use Utopia\App;
+use Utopia\CLI\Console;
+use Utopia\Config\Config;
+use Utopia\Database\Database;
+use Utopia\Database\DateTime;
+use Utopia\Database\Query;
+use Utopia\Platform\Action;
+use Utopia\Queue\Client;
+use Utopia\System\System;
+
+class EdgeSync extends Action
+{
+    public static function getName(): string
+    {
+        return 'edge-sync';
+    }
+
+    public function __construct()
+    {
+        $this
+            ->desc('Schedules edge sync tasks')
```

Comment:
Why do we use a scheduler? I feel this will not be good enough for production. We need to strive to keep sync latency below 300ms at worst. This intervals might not cut it. I thought we will have a long running tasks that constantly fetches logs and send them async to all regions at the same time and has some retry logic.